modules/olm-operatorgroups-copied-csvs.adoc

```diff
@@ -10,3 +10,8 @@ OLM creates copies of all active member CSVs of an Operator group in each of the
 Copied CSVs have a status reason `Copied` and are updated to match the status of their source CSV. The `olm.targetNamespaces` annotation is stripped from copied CSVs before they are created on the cluster. Omitting the target namespace selection avoids the duplication of target namespaces between tenants.
 
 Copied CSVs are deleted when their source CSV no longer exists or the Operator group that their source CSV belongs to no longer targets the namespace of the copied CSV.
+
+[NOTE]
+====
+The `disableCopiedCSVs` toggle must be disabled. After disabling a `disableCopiedCSVs` toggle, the OLM deletes existing copied CSVs on a cluster. When a `disableCopiedCSVs` toggle is enabled, the OLM adds Copied CSVs again.
```

Comment:
@xenolinux 
```console
The `disableCopiedCSVs` toggle must be disabled.
```
 better be 
```console
The `disableCopiedCSVs` toggle is disabled by default.
```

About
```console
After disabling a `disableCopiedCSVs` toggle, the OLM deletes existing copied CSVs on a cluster. When a `disableCopiedCSVs` toggle is enabled, the OLM adds Copied CSVs again.
```
it is not correct (sorry for the first time not to identify it) because disabling `disableCopiedCSVs` seems to open it.
actually if we want to enable `disableCopedCSVs`, we need to set
```yaml
apiVersion: operators.coreos.com/v1
kind: OLMConfig
metadata:
  name: cluster
spec:
  features:
    disableCopiedCSVs: true
```
and if we want to disable `disableCopedCSVs`, we need to set
```yaml
apiVersion: operators.coreos.com/v1
kind: OLMConfig
metadata:
  name: cluster
spec:
  features:
    disableCopiedCSVs: false
```

so, I suggest to change to 
```console
After enabling a `disableCopiedCSVs` toggle, the OLM deletes existing copied CSVs on a cluster. When a `disableCopiedCSVs` toggle is disabled, the OLM adds Copied CSVs again.
Disabling the toggle:
```yaml
apiVersion: operators.coreos.com/v1
kind: OLMConfig
metadata:
  name: cluster
spec:
  features:
    disableCopiedCSVs: false

Enabling the toggle:
```yaml
apiVersion: operators.coreos.com/v1
kind: OLMConfig
metadata:
  name: cluster
spec:
  features:
    disableCopiedCSVs: false
```
```

Reply:
Ack'ed